modules/dnn/test/test_onnx_importer.cpp

```diff
@@ -2622,24 +2623,35 @@ static void testYOLO(const std::string& weightPath, const std::vector<int>& refC
 
     Mat img = imread(imgPath);
 
-    Mat inp = blobFromImageWithParams(img, imgParams);
+    Mat inp = blobFromImageWithParams(img, imgParams);  
```

Comment:
CI blames a trailing whitespace

Reply:
White spaces are NOT detected locally, not do break the build. Additionally, I do not have CI running locally. So it quite easy to forget about white spaces. Sorry for inconvenience.

Reply:
You may setup a related setting in IDE. For example, VSCode: https://stackoverflow.com/a/30884298

Reply:
> White spaces are NOT detected locally

Again, mentioned Wiki page has BKM how to enable that.